libs/island-ui/core/src/index.ts

```diff
@@ -58,6 +58,8 @@ export * from './lib/PdfViewer/PdfViewer'
 export * from './lib/PageLoader'
 export * from './lib/VisuallyHidden/VisuallyHidden'
 export * from './lib/Drawer/Drawer'
+export * from './lib/ProblemTemplate/ProblemTemplate'
+export * from './lib/ProblemTemplate/ProblemTemplate.css'
```

Comment:
We don't need to export the css? Unless I missed something 🙈

Reply:
This is specifically imported in NoData component